apps/src/StudioApp.js

```diff
@@ -1044,47 +1078,49 @@ StudioApp.prototype.setConfigValues_ = function (config) {
 StudioApp.prototype.configureDom_ = function (config) {
   var container = document.getElementById(config.containerId);
   container.innerHTML = config.html;
-  var runButton = container.querySelector('#runButton');
-  var resetButton = container.querySelector('#resetButton');
-  var throttledRunClick = _.debounce(this.runButtonClick, 250, true);
-  dom.addClickTouchEvent(runButton, _.bind(throttledRunClick, this));
-  dom.addClickTouchEvent(resetButton, _.bind(this.resetButtonClick, this));
-
-  var belowViz = document.getElementById('belowVisualization');
-  var referenceArea = document.getElementById('reference_area');
-  if (referenceArea) {
-    belowViz.appendChild(referenceArea);
-  }
+  if (this.usesStandardLayout_) {
```

Comment:
Sounds good.  I considered not calling this method at all in that case, but the first couple of lines which actually inject the app HTML are important - maybe they belong in a separate method?